docs/build_from_source.md

```diff
@@ -46,19 +52,39 @@ development Docker image.
 development Docker image.
 
     ``` bash
+    # For GPU backend
     docker run --gpus all --rm -it -v $PWD:/disc bladedisc/bladedisc:latest-devel-cuda11.0 bash
+
+    # For CPU backend
+    docker run --rm -it -v $PWD:/disc bladedisc/bladedisc:latest-devel-cuda11.0 bash
```

Comment:
The commands are the same as for TensorFlow Users
We may move "docker run --rm -it -v $PWD:/disc bladedisc/bladedisc:latest-devel-cuda11.0 bash" to line19.

Thus this passage is no longer needed to be duplicated.
"""
**Note that we use the same development docker images for both Nvidia GPU backend and X86 CPU backend. For X86 CPU backend, both the building phase and execution phase do not require CUDA available.**
"""

Reply:
done